qubes-rpc/admin.repos.Disable

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+
+# Empty output indicates success; any input indicates error (probably an exception)
+
+import dnf
+import iniparse
+import sys
+
+base = dnf.Base()
+
+base.read_all_repos()
+
+reponame = sys.stdin.readline()
```

Comment:
Better use service argument (`sys.argv[1]`), as it will allow policing specific repositories (for example allow to enable/disable current-testing, but not security-testing).